spec/views/content/shared/funding.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+describe "_funding.html.erb" do
+  let(:subject_name) { "chemistry" }
+  let(:salary_value) { YAML.load_file("config/values/salaries.yml").dig("burseries", "postgraduate", "chemistry") }
```

Comment:
```suggestion
  let(:salary_value) { YAML.load_file("config/values/salaries.yml").dig("bursaries", "postgraduate", "chemistry") }
```
Typo - not sure how it affects the test though!